composites/Plugin/SnippetEditor/components/ReplacementVariableEditorStandalone.js

```diff
@@ -345,9 +346,22 @@ class ReplacementVariableEditorStandalone extends React.Component {
 	 * @returns {void}
 	 */
 	setEditorRef( editor ) {
+		try {
+			editor.editor.editorContainer.firstChild.id = this.props.fieldId;
+		} catch ( error ) {
+			console.trace( error );
+		}
+
 		this.editor = editor;
 	}
 
+	setEditorFieldId() {
+		const editorContainer = get( this.editor, "editor.editorContainer" );
+		const editorField = editorContainer.querySelector( "div[contenteditable]" );
```

Comment:
Unsure about this, but I believe the equal signs need to be aligned if we want to adhere to the codestyle.